### PR TITLE
added User#asMention

### PR DESCRIPTION
**Pull request by tiagodinis33**
**Please describe the changes this PR makes and why it should be merged:**
I added User#asMention that returns a mention that can be displayed on a message, because its more intuitive
example:
```js
msg.channel.send(`Hi ${msg.author.asMention}!`)
```
will send:
```Hi @tiagodinis33!```
**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

File: src/structures/User.js [hunk 1]
```diff
@@ -91,9 +92,14 @@ class User extends Base {
        * @type {?UserFlags}
        */
       this.flags = new UserFlags(data.public_flags);
+      
     }
   }
 
+  get asMention(){
+    return `<@${this.id}>`;
+  }
```
Comment (shinotheshino) [inline src/structures/User.js:101]: Isn't this the exact same as `User#toString()`?

**Comment by GoldenAngel2** [comment]
`User.toString()` does this.

File: src/structures/User.js (hunk 1 above)
Comment (tiagodinis33) [reply src/structures/User.js:101]: wait, toString already do that?

Comment (tiagodinis33) [reply src/structures/User.js:101]: i thougth it would return the object as string

Comment (GoldenAngel2) [reply src/structures/User.js:101]: Yes, it does.

Comment (shinotheshino) [reply src/structures/User.js:101]: https://github.com/discordjs/discord.js/blob/5ca97c93515d4dfaa2b4951a020abc000115ed4d/src/structures/User.js#L248

Comment (tiagodinis33) [reply src/structures/User.js:101]: i mean, return a json

Comment (tiagodinis33) [reply src/structures/User.js:101]: but anyway

Comment (tiagodinis33) [reply src/structures/User.js:101]: thx for your attention

**Pull request closed without merging**